system/jlib/jutil.hpp

```diff
@@ -360,10 +360,18 @@ extern jlib_decl bool getConfigurationDirectory(const IPropertyTree *dirtree, //
                                                 const char *instance, 
                                                 StringBuffer &dirout);
 
-extern jlib_decl bool querySecuritySettings(bool *          _useSSL,
+extern jlib_decl bool querySecuritySettings(SSLCfg *        _useSSL,
                                             unsigned short *_port,
                                             const char * *  _certificate,
-                                            const char * *  _privateKey);
+                                            const char * *  _privateKey,
+                                            const char * *  _passPhrase);
+
+extern jlib_decl bool queryDafsServerSecSettings(SSLCfg *        _useSSL,
+                                                 unsigned short *_port,
+                                                 unsigned short *_sslport,
+                                                 const char * *  _certificate,
+                                                 const char * *  _privateKey,
+                                                 const char * *  _passPhrase);
```

Comment:
The key is that dafilesrv can set both and might use both, but clients receive this info from what is configured.  Clients therefore only get ONE port back by design.